frontend/src/lib/k8s/pod.ts

```diff
@@ -19,6 +24,7 @@ export interface KubePodSpec {
   readinessGates?: {
     conditionType: string;
   }[];
+  volume: KubeVolume[];
```

Comment:
This property is optional and it's called `volumes`. After this change, the error with the storyHelper should be fixed.